servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingHttpService.java

```diff
@@ -40,7 +40,8 @@ HttpResponse handle(HttpServiceContext ctx, HttpRequest request, HttpResponseFac
 
     @Override
     default HttpExecutionStrategy requiredOffloads() {
-        return OFFLOAD_NONE_STRATEGY;
+        // safe default--implementations are expected to override
+        return OFFLOAD_RECEIVE_DATA_STRATEGY;
```

Comment:
@bondolo I'm surprised that there are no tests that could catch cases like this one. Can you please have a look how this value is used? Is it used for computation of the final strategy?

Reply:
Indeed, I would have expected that the existing tests would have caught this.

Reply:
It is referenced by the some of the Jersey tests but seems to have no effect. I will continue to investigate why.

Reply:
We can work on tests in a follow-up. I would like to pull in changes for interfaces, bcz I'm working on filters API